example/package.json

```diff
@@ -3,6 +3,20 @@
     "eslint": "*",
     "@typescript-eslint/parser": "*",
     "@typescript-eslint/eslint-plugin": "*",
-    "typescript": "5.0"
+    "typescript": "5.0",
+    "prettier": "^2.8.7",
+    "prettier-plugin-sh": "^0.12.8",
+    "prettier-plugin-sql": "^0.14.0"
+  },
+  "//FIXME(alexeagle)": "This section shouldn't be needed since these are plugins",
```

Comment:
not needed anymore since you're public hoisting?

Reply:
I couldn't get it to work without this... as usual

Reply:
come help me later after merge 😁